### PR TITLE
DS-1583 fix: Allow CTA Menu Block to reference existing blocks

### PR DESCRIPTION
**Pull request by froboy**
To test:

- go to `/admin/structure/menu/manage/main`
- edit a menu item
- observe "Add existing content block"
- 
SCR-20240724-pbcv

### CODE REVIEW

File: modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+/**
+ * @file
+ * Install, update and uninstall functions for the Website Services Main Menu CTA block module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function y_lb_main_menu_cta_block_install(): void {
+  \Drupal::messenger()->addStatus(t('Module Website Services Main Menu CTA block has been installed.'));
```
Comment (podarok) [inline modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install:14]: Why do we need this?
Core is doing such messages
image

File: modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function y_lb_main_menu_cta_block_uninstall(): void {
+  \Drupal::messenger()->addStatus(t('Module Website Services Main Menu CTA block has been uninstalled.'));
```
Comment (podarok) [inline modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install:21]: No need for this 
image

**Review by podarok — CHANGES_REQUESTED** [review]
remove redundand logs, please

File: modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install (hunk 1 above)
Comment (froboy) [reply modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install:14]: Those were from `drush generate install-file` and I'd just left them. Thanks for the feedback.

**Pull request merged**